streams/examples/main/java/org/apache/kafka/streams/examples/wordcount/WordCountJob.java

```diff
@@ -27,21 +24,24 @@
 import org.apache.kafka.common.serialization.Serializer;
 import org.apache.kafka.common.serialization.StringSerializer;
 import org.apache.kafka.common.serialization.StringDeserializer;
-import org.apache.kafka.connect.json.JsonSerializer;
 import org.apache.kafka.streams.KafkaStreams;
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.StreamsConfig;
-import org.apache.kafka.streams.kstream.Aggregator;
 import org.apache.kafka.streams.kstream.KStreamBuilder;
 import org.apache.kafka.streams.kstream.KStream;
+import org.apache.kafka.streams.kstream.KTable;
 import org.apache.kafka.streams.kstream.KeyValueMapper;
-import org.apache.kafka.streams.kstream.UnlimitedWindows;
 import org.apache.kafka.streams.kstream.ValueMapper;
-import org.apache.kafka.streams.kstream.Windowed;
 
 import java.util.Arrays;
 import java.util.Properties;
 
+/**
+ * Implements the "WordCount" program that computes a simple word occurrence histogram over text files.
```

Comment:
"over text files": This is confusing because we're not using text files anywhere.

What about the following:

> Implements the WordCount program that computes a simple word occurrence histogram from an input text.
> Assumes the input text is read from the Kafka topic "streams-input-lines", where the values of messages represent lines of text.